frontend/src/components/admin/AddAdminModal.tsx

```diff
@@ -40,7 +40,9 @@ const AddAdminModal = ({
 
   const [isLoading, setIsLoading] = useState(false);
 
-  const [createAdminInvite] = useMutation(CREATE_ADMIN_INVITE);
+  const [createAdminInvite] = useMutation(CREATE_ADMIN_INVITE, {
+    refetchQueries: ["GET_USER_INVITE"],
```

Comment:
```suggestion
    refetchQueries: ["UserInvitesModal_getUserInvites"],
```

right idea but referring to the wrong graphql query name

Reply:
thanks for feedback!